docs/pages/desktop-access/getting-started.mdx

```diff
@@ -25,78 +25,98 @@ This guide requires you to have:
+
+Find your domain in the side menu, and right click the container you want the service account to live in (typically Users). From the context
+menu, select `New > User` and fill out the fields for your Teleport LDAP Service Account. For the "User logon name" field, you can use `svc-teleport`
+On the password screen, unselect  “User must change password at next login” and select “Password never expires”. Make a note of what values you put 
+for user logon name and password, which will be used in your `teleport.yaml` configuration file later, and then click through the rest of the wizard.
 
 <Admonition
   type="note"
-  title="Teleport CA"
+  title="Default User Permissions"
 >
-  Prior to v8.0, the Teleport CA was not compatible with Windows logins. If
-  you're setting up Desktop Access in an existing cluster created before v8.0,
-  you must first perform a [CA rotation](../setup/operations/ca-rotation.mdx)
-  in order to resolve this.
+  By default, domain users have Read and List permissions throughout Active Directory. If you have configured
+  any restrictions to Domain Users, you may have to adjust which groups this account is a member of.
 </Admonition>
 
-First, we need to enable Desktop Access in Teleport. To do this, add the
-following section in `teleport.yaml` on your Linux server:
+<Figure align="left" bordered caption="Create Service Account">
+  ![Create Service Account](../../img/desktop-access/ad-new-user.png)
+</Figure>
 
-```yaml
-windows_desktop_service:
-  enabled: yes
-  # This is the address that windows_desktop_service will listen on.
-  listen_addr: "0.0.0.0:3028"
-  # (optional) This is the address that windows_desktop_service will advertise
-  # to the rest of Teleport for incoming connections. Only proxy_service should
-  # connect to windows_desktop_service, users connect to the proxy's web UI
-  # instead.
-  public_addr: "desktop-access.example.com:3028"
-  ldap:
-    # Address of the Domain Controller for LDAP connections. Usually, this
-    # address will use port 389, like: domain-controller.example.com:389.
-    addr:     '$LDAP_SERVER_ADDRESS'
-    # Active Directory domain name you are connecting to.
-    domain:   '$LDAP_DOMAIN_NAME'
-    # LDAP username for authentication. This username must include the domain
-    # NetBIOS name.
-    #
-    # For example, if your domain is "example.com", the NetBIOS name for it is
-    # likely "EXAMPLE". When connecting as the "Administrator" user, you should
-    # use the format: "EXAMPLE\Administrator".
-    username: '$LDAP_USERNAME'
-    # Plain text file containing the LDAP password for authentication.
-    # This is usually the same password you use to login to the Domain Controller.
-    password_file: /var/lib/ldap-pass
-```
+Now right click the same container you did before, and select `New > Group`. Choose a Group name like `svc-teleport-group` and ensure that “Global”
```

Comment:
Observation: users will be making a lot of mistakes here. Can we implement something like https://github.com/gravitational/teleport/pull/8896 for Desktop Access?

Reply:
We have a separate issue filed to look into how much of this we can automate. It's looking like the answer is "not very much" but we'll see what we can do.